builds/misc/images-release-base-image-update.yaml

```diff
@@ -0,0 +1,770 @@
+trigger: none
+pr: none
+
+name: $(version)
+
+stages:
+# This stage builds Rocks DB and adds the files to staging directory
+################################################################################
+- template: templates/build-rocksdb.yaml
```

Comment:
Are these steps common enough between this and images-release.yaml that we can make a template?

Reply:
There's enough in common to at least warrant looking at making a template. The reason it's a separate file is because we are introducing a new process, which might not go according to plan because it's not vetted yet.

If I create a template, then I risk breaking the existing release process too. Far better to let the churn settle and then refactor.